core-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -1943,6 +1943,66 @@
       <trans-unit id="SECTIONS.manual">
         <source>Manual output</source>
       </trans-unit>
+      <trans-unit id="DCA.all.0">
+        <source>Edit multiple</source>
+      </trans-unit>
+      <trans-unit id="DCA.all.1">
+        <source>Edit multiple elements at once</source>
+      </trans-unit>
+      <trans-unit id="DCA.toggleNodes.0">
+        <source>Toggle all</source>
+      </trans-unit>
+      <trans-unit id="DCA.toggleNodes.1">
+        <source>Expand or collapse all nodes</source>
+      </trans-unit>
+      <trans-unit id="DCA.new.0">
```

Comment:
Feels strange that `new` and a few other have to be an array, but most other operations do not?

Reply:
Global operations have both a title and a link text and are therefore arrays. Regular operations have only a title (and an icon instead of a link text) and are therefore strings.